src/music.cpp

```diff
@@ -149,7 +149,7 @@ bool MusicSet::FillSetDetails(IniFile *ini, const char *path, const char *full_f
 				this->songinfo[i].filetype = MTT_STANDARDMIDI;
 			}
 
-			const char *trimmed_filename = filename;
+			const char *trimmed_filename = filename.c_str();
```

Comment:
Worth converting usage of `trimmed_filename` as well? Could use `std::string_view`

Reply:
It wouldn't get much prettier. Primarily because `find` returns `npos`, so to iterate to the next you must first `find`and then compare the result to determine how to change the `string_view` meaning it's not a simple for loop anymore. Or am I missing something simpler that we could do?
I'd almost be tempted not keep a pointer, but rather the index into the string. Though that messes with the rest of the function where the pointer is still used.
So I rather leave that conversion/significant rewrite to another PR.